### PR TITLE
fix: Scrolling causes the crypto icons to clip over the banner

### PR DESCRIPTION
**Pull request by irohdw**
The need to add /fox-farm to the url to avoid a 404 error has been added to the README.

The need to run the command rm -f ./npmrc to fix the error "Failed to replace env in config ${NPM_TOKEN}" has been added to the README.

### CODE REVIEW

**Review by reidrankin — APPROVED** [review]

**Comment by reidrankin** [comment]
approved, pending a check that the removal of .npmrc doesn't break CI

**Review by 0xean — APPROVED** [review]

**Comment by 0xean** [comment]
conditionally approved waiting for @reidrankin confirmation so we can pay out the bounty!

**Comment by reidrankin** [comment]
Unfortunately, I didn't set up the CI for this and have no Cloudflare access to check how it works. I think maybe @con5cience did? We either need to check before merge to make sure the `.npmrc` won't break anything -- or maybe just #sendit and be ready to revert if it does

**Pull request merged**
